docs/src/user-guide/guides-using-object-storage/compress.md

```
@@ -0,0 +1,80 @@
+* `<credentials-file>` is the path to an AWS credentials file like the following:
+
+    ```ini
+    [default]
+    aws_access_key_id = <aws-access-key-id>
+    aws_secret_access_key = <aws-secret-access-key>
+    ```
+
+    * CLP expects the credentials to be in the `default` section.
+    * `<aws-access-key-id>` and `<aws-secret-access-key>` are the access key ID and secret access
+      key of the CLP IAM user.
+    * If you don't want to use a credentials file, you can specify the credentials on the command
```

Markdownlint (0.37.0)

Expected: 2; Actual: 4
Unordered list indentation

(MD007, ul-indent)